quint/apalache-tests.md

```diff
@@ -1,74 +1,75 @@
 # Integration tests against Apalache
 
 
-All of the test inputs in the following test cases are commands executed by `bash`.
+All of the test inputs in the following test cases are commands executed by
+`bash`.
+
+We currently assume that the Apalache server has already been started. 
+So before running these tests, in a separate terminal session, run
+
+```
+apalache-mc server
+```
+
+This requirement will be removed with https://github.com/informalsystems/quint/issues/823
 
 <!-- !test program
-PATH=_build/apalache/bin:$PATH bash -
+APALACHE_DIST=_build/apalache PATH=_build/apalache/bin:$PATH bash -
 -->
 
+## Configuration errors
 
-## Can check `../examples/language-features/booleans.qnt`
+### Verifying spec with invalid init param produces an error
 
-<!-- !test in can check booleans.qnt -->
+<!-- !test in invalid init -->
 ```
-quint typecheck --out _build/booleans.qnt.json ../examples/language-features/booleans.qnt
-apalache-mc check _build/booleans.qnt.json | grep -o "EXITCODE: OK"
+quint verify --init=invalidInit ../examples/language-features/booleans.qnt
 ```
 
-<!-- !test out can check booleans.qnt -->
+<!-- !test exit 1 -->
+<!-- !test err invalid init -->
 ```
-EXITCODE: OK
+error: Configuration error (see the manual): Operator invalidInit not found (used as the initialization predicate)
 ```
 
-## Can check `../examples/language-features/integers.qnt`
 
-<!-- !test in can check integers.qnt -->
-```
-quint typecheck --out _build/integers.qnt.json ../examples/language-features/integers.qnt
-apalache-mc check _build/integers.qnt.json | grep -o "EXITCODE: OK"
-```
+## Successful verification
```

Comment:
The following tests are just mechanical replacement of the manual cli invocations with invocation of the `verify` command.